### DISCUSSION TITLE
各位同行，下周有位警察局的阿Sir要来我家，不会发现我在绳网揽生意吧

### DISCUSSION THREAD

**Discussion by lingyushen** [share121/inter-knot#993] [其他]
![1c02f3fbbe6c8f33f68b0a181eaaac44](https://github.com/user-attachments/assets/43c65e6c-55c5-439a-a025-0c0270ef8a76)

**Comment by gaojiaobei** [comment]
建议赶紧开溜，举报绳匠可是有奖励的，你说不定被熟人或者客户卖了！
